You're failing pep8.
```
2017-03-20 19:35:35 ERROR: Found 1 pep8 issue(s) which need to be resolved:
2017-03-20 19:35:35 ERROR: lib/ansible/modules/cloud/amazon/s3.py:717:77: W291 trailing whitespace (100%)
```
If you fix that I'll merge this and close mine.

--- a/lib/ansible/modules/cloud/amazon/s3.py
+++ b/lib/ansible/modules/cloud/amazon/s3.py
@@ -713,7 +713,7 @@ def main():
 def get_s3_connection(aws_connect_kwargs, location, rgw, s3_url):
     if s3_url and rgw:
         rgw = urlparse(s3_url)
-        # ensure none of the named arguments we will pass to boto.connect_s3 
+        # ensure none of the named arguments we will pass to boto.connect_s3
         # are already present in aws_connect_kwargs
         for kw in ['is_secure', 'host', 'port', 'calling_format']:
             try:

